plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -62,7 +62,8 @@ func calculateOccupancy(pod *api.Pod, node api.Node, pods []*api.Pod) algorithm.
 
 	cpuScore := calculateScore(totalMilliCPU, capacityMilliCPU, node.Name)
 	memoryScore := calculateScore(totalMemory, capacityMemory, node.Name)
-	glog.V(10).Infof(
+//	glog.V(10).Infof(
```

Comment:
Please revert and probably change V(10) to V(4) or something more reasonable than 10.